sentry-graphql/src/main/java/io/sentry/graphql/SentryInstrumentation.java

```diff
@@ -174,8 +200,7 @@ public CompletableFuture<ExecutionResult> instrumentExecutionResult(
                     // not capturing INTERNAL_ERRORS as they should be reported via graphQlContext
```

Comment:
We could move that comment into the new `isIgnored` method, or update/remove it.